src/cmd_kv.cc

```diff
@@ -215,4 +215,35 @@ void BitCountCmd::DoCmd(PClient* client) {
   client->AppendInteger(static_cast<int64_t>(count));
 }
 
+IncrCmd::IncrCmd(const std::string& name, int16_t arity)
+    : BaseCmd(name, arity, CmdFlagsReadonly, AclCategoryRead | AclCategoryString) {}
+
+bool IncrCmd::DoInitial(pikiwidb::PClient *client) {
```

Comment:
```suggestion
bool IncrCmd::DoInitial(pikiwidb::PClient* client) {
```
注意代码风格与项目一致，排查下相关问题吧